saksbehandler/server/src/config.ts

```diff
@@ -1,48 +1,17 @@
-import logger from './logger';
-
-export const envVar = ({ name, required = true }: { name: string; required?: boolean }) => {
+const envVar = ({ name, required = true }: { name: string; required?: boolean }) => {
     if (!process.env[name] && required) {
-        logger.error(`Missing required environment variable '${name}'`);
+        console.error(`Missing required environment variable '${name}'`);
```

Comment:
Vil ikke denne endringen brekke loggformateringen i kibana? Da mister vi isåfall loggnivået og risikerer at meldinger over flere linjer blir separate logginnslag.

Reply:
Det vet jeg ikke. Jeg sjekket at logging i podden såg bra ut. Men jeg skjønte ikke helt hvorfor vi brukte `winston`? 🤔 Vi brukte bare console-logging i tiltaksgjennomføring sin server.

Reply:
Transporteren til Winston var bare `new winston.transports.Console()`, så det virker jo som om den bare logger i consolen uansett 🤷